datalad_dataverse/remote.py

```diff
@@ -120,6 +134,9 @@ def transfer_retrieve(self, key, file):
         with open(file, "wb") as f:
             f.write(response.content)
 
+    def transferexport_retrieve(self, key, local_file, remote_file):
+        self.transfer_retrieve(remote_file, local_file)
```

Comment:
Minor remark on occasion of that line (but true for other calls to the non-export methods as well):
I think argument assignments can turn pretty confusing for someone having to occasionally maintain this code. I think I'd prefer using keywords to be explicit here - `self.transfer_retrieve(key=remote_file, file=local_file)`

Reply:
Done